packages/truffle-deployer/test/deployer.js

```diff
@@ -236,8 +241,8 @@ describe("Deployer (sync)", function() {
 
     utils.stopAutoMine();
 
-    const isLibrary = await IsLibrary.deployed();
-    const example = await Example.deployed();
+    await IsLibrary.deployed();
+    await Example.deployed();
```

Comment:
We can just remove these lines I think if we're not using the results.

Reply:
Alright, I usually do "as minimum changes as possible" and not "drive by cleanup", but I can remove these if you really want me to.

Reply:
The only reason these were altered to begin with is because of the rolling linting changes required everytime I modify a file that hasn't been through the linting process yet

Reply:
I'm going to go ahead and do the drive-by cleanup to not have to wait for another cycle of discussion. I'm usually not fond of this, but will make an exception.